src/control/index.ts

```diff
@@ -1,34 +1,29 @@
 // Index Operations
-export { configureIndex, type ConfigureIndexOptions } from './configureIndex';
-export { createIndex, type CreateIndexOptions } from './createIndex';
-export { deleteIndex, type DeleteIndexOptions } from './deleteIndex';
-export {
-  describeIndex,
-  type DescribeIndexOptions,
-  type IndexDescription,
-} from './describeIndex';
-export {
-  listIndexes,
-  type IndexList,
-  type PartialIndexDescription,
-} from './listIndexes';
+export { configureIndex } from './configureIndex';
```

Comment:
These formatting changes don't read nearly as nice, but are necessary for compatibility with older versions of typescript.